src/tests/build.sh

```diff
@@ -121,7 +121,7 @@ build_Tests()
     buildArgs+=("/p:NUMBER_OF_PROCESSORS=${__NumProc}")
     buildArgs+=("${__UnprocessedBuildArgs[@]}")
 
-    # Disable warnAsError - coreclr issue 19922
+    # Disable warnAsError - https://github.com/dotnet/runtime/issues/11077
```

Comment:
As this issue has been closed, it's worth taking a look and seeing if we can remove the warnAsError flag.

Reply:
Removed in the latest commit, let's see what the lab says.

Reply:
Reverted per your PR comment, I'm rerunning the outerloop job.